lib/std/io/tty.zig

```diff
@@ -5,6 +5,46 @@ const process = std.process;
 const windows = std.os.windows;
 const native_os = builtin.os.tag;
 
+/// Registers a global handler function to run if an abort signal is catched.
+/// An abort signal is usually fired if Ctrl+C is pressed in a terminal.
+///
+/// Because this handler won't run if Ctrl+C isn't pressed by the user, registering this handler failed,
+/// or the handler was overwritten by another setAbortSignalHandler call,
+/// this should be used only for non-critical cleanups or resets of terminal state and such.
+///
+/// A program can only have one handler at a time.
+///
+/// The handler will not exit the program after it runs.
+pub fn setAbortSignalHandler(comptime handler: *const fn () void) error{Unexpected}!void {
```

Comment:
I really don't like the name when `SIGABRT` exists.

```suggestion
pub fn setInterruptSignalHandler(comptime handler: *const fn () void) error{Unexpected}!void {
```

Reply:
I guess this is fine.

Reply:
Yeah probably better.